### DISCUSSION TITLE
rename and sort custom columns

### DISCUSSION THREAD

**Discussion by lublak** [Budibase/budibase#7452] [Help] [answered]
I can create custom columns with budibase. But how can i rename it and sort it? And how can i add multiple coumns.
![grafik](https://user-images.githubusercontent.com/44057030/186613437-f9110530-2423-47d8-84cd-17d5d418419a.png)
![grafik](https://user-images.githubusercontent.com/44057030/186612597-1693dfe3-0460-42e3-8d82-c9abb5845c76.png)

**Comment by R2bEEaton** [comment, marked as answer]
Hi @lublak,

What you're describing is not currently possible. Custom columns are not sortable or namable, and you can't add multiple. This column is meant for buttons or very simple elements, example below.

You can achieve the _look_ you're going for (or at least something similar) by setting the widths of your elements using custom CSS and inserting a vertical Divider component. You can even get crazier with custom CSS in an Embed component to make it look more like the real table.

![image](https://user-images.githubusercontent.com/34921506/186646390-002f76f6-1dc5-4c94-92dd-f43a715464a4.png)
![image](https://user-images.githubusercontent.com/34921506/186646426-8b6b04c2-7e88-4eb2-b80d-bfb88afa4677.png)
This is an example of what the custom column table is really meant for, but that doesn't stop you from customizing Budibase until you're content!

Sorry this wasn't more helpful. Reply with any questions or mark this as answered with the checkbox above to help future readers!

**Reply by lublak** [reply]
Thank you for the answer. It's not perfect, of course, but it's better than nothing :) it's just a bit confusing to work with.
